src/beautifiers/beautysh.coffee

```diff
@@ -0,0 +1,15 @@
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class BashBeautify extends Beautifier
+  name: "beautysh"
+  link: "https://github.com/bemeurer/beautysh"
+  options: {
+    Bash:
+      indent_size: true
```

Comment:
This is great!

I think all that is missing is to update the docs a final time so `indent_size` shows up under available options 😄 .